dhis2-mobile-program-rules/src/main/java/org/dhis2/mobileProgramRules/RulesRepository.kt

```diff
@@ -335,7 +335,7 @@ class RulesRepository(private val d2: D2) {
             dueDate = event.dueDate()?.toRuleEngineLocalDate(),
             completedDate = event.completedDate()?.toRuleEngineLocalDate(),
             organisationUnit = event.organisationUnit()!!,
-            organisationUnitCode = d2.organisationUnit(event.organisationUnit()!!)?.code()!!,
+            organisationUnitCode = d2.organisationUnit(event.organisationUnit()!!)?.code() ?: "",
```

Comment:
`organisationUnitCode` is type `String?`, why can not be null juts`...?.code()`

Reply:
in this case for this class it is not